Framework/Json/CMakeLists.txt

```
@@ -0,0 +1,38 @@
+set(SRC_FILES
+    src/Json.cpp
+)
+
+set(INC_FILES
+    inc/MantidJson/Json.h
+)
+
+set (TEST_FILES
+    test/JsonTest.h
+)
+
+add_library(Json ${SRC_FILES} ${INC_FILES})
+target_link_libraries(Json ${JSONCPP_LIBRARIES})
```

```suggestion
target_link_libraries(Json PUBLIC ${JSONCPP_LIBRARIES})
```
to make it clear this is part of the public library interface.